test/test_helper.js

```diff
@@ -79,6 +79,20 @@ class TestHelper {
         };
     };
 
+    fakeUserWithServiceTermsAccepted = () => {
+        return {
+            ...this.fakeUserWithId(),
+            accepted_service_terms_id: 1,
+        };
+    };
+
+    fakeUserWithServiceTermsNotAccepted = () => {
+        return {
+            ...this.fakeUserWithId(),
+            accepted_service_terms_id: 0,
+        };
+    };
```

Comment:
I don't think these really need to be functions in the test helper since they're so specific. I'd suggest just doing this in-line where needed or create helper functions right in the test file itself if you still want functions